<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de diagnostico IRA</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 246, 255);
background-color: rgb(243, 246, 255);
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>281</width>
     <height>21</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial&quot;;
font: 75 16pt &quot;Arial&quot;;
font: 12pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>SISTEMA INFECCIONES RESPIRATORIAS AGUDAS </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>PACIENTE</string>
   </property>
   <widget class="QPushButton" name="ipaciente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="epaciente">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="diagnostico1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Diagnostico</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>CONSULTAS</string>
   </property>
   <widget class="QPushButton" name="historial">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Historial</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grafica">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Grafica IRA</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="salir">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>lungs-2803208_960_720.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>salir</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
